src/MessagePack.GeneratorCore/Generator/StringKey/StringKeyFormatterDeserializeHelper.cs

```diff
@@ -31,6 +31,11 @@ public static string Classify(ObjectSerializationInfo objectSerializationInfo, s
 
         private static bool IsConstructorParameter(ObjectSerializationInfo objectSerializationInfo, MemberSerializationInfo member)
         {
+            if (objectSerializationInfo.HasInitOnlySetter)
```

Comment:
dont need '___VarName__initialized' flags, then return true when HasInitOnlySetter is as well as for the case matching constructor parameter.